src/jquery.mask.js

```diff
@@ -452,9 +452,9 @@
         return this;
     };
 
-    $.fn.unmask = function() {
-        clearInterval($.maskWatchers[this.selector]);
-        delete $.maskWatchers[this.selector];
+    $.fn.unmask = function(selector) {
+        clearInterval($.maskWatchers[this.selector||selector]);
```

Comment:
Extract this to a variable and re-use it. Also... separate the `||` with spaces.